microsoft-edge/dev-videos/index.md

```diff
@@ -30,265 +30,816 @@ To add a new video:
-## Web Platform - Create a scroll-linked animation without JavaScript
+See also:
+* [Debug DOM memory leaks with the Detached Elements tool](../devtools-guide-chromium/memory-problems/dom-leaks.md)
+* [Debug memory leaks with the Microsoft Edge Detached Elements tool](https://blogs.windows.com/msedgedev/2021/12/09/debug-memory-leaks-detached-elements-tool-devtools/) - blog post.
 
-_Published on 12 May 2022._
 
-[![Thumbnail image for the scroll-linked animation API video](./index-images/scroll-linked-animations.png)](https://www.youtube.com/watch?v=Q0nhiHVVnvI)
+<!-- ====================================================================== -->
+## Web Platform - The JSON viewer
 
-Learn about the upcoming CSS scroll-linked animations feature and how it can be used to create a reading progress indicator on a webpage, without using JavaScript.
+August 17, 2023
 
-CSS scroll-linked animations is an experimental feature in Microsoft Edge.  To try this feature, go to `edge://flags` and then enable the **Experimental Web Platform features** setting.
+https://www.youtube.com/watch?v=DO7V6lw1SWA
 
-To play with the demo application that's shown in the video, see the rendered [reader demo](https://microsoftedge.github.io/Demos/reader/) and its [source code](https://github.com/MicrosoftEdge/Demos/tree/main/reader).
+The JSON viewer automatically formats and highlights JSON responses and files in browser tabs.  When your web server responds to HTTP requests with data encoded as JSON, this data isn't always easy to read and is sometimes returned as one line of text.  The JSON viewer changes the returned data to make it easier to read.  The JSON syntax is highlighted with different colors, object properties are displayed on their own lines and indented, and objects can be collapsed or expanded.
 
-To learn more about the CSS scroll-linked animations feature, see [@scroll-timeline](https://developer.mozilla.org/docs/Web/CSS/@scroll-timeline) at MDN.
+See also:
+* [View formatted JSON](../devtools-guide-chromium/json-viewer/json-viewer.md)
 
 
 <!-- ====================================================================== -->
-## DevTools - Customizing Microsoft Edge Developer Tools and quick feature access
+## Web Platform - Fully style the drop-down part of a `<select>` with the new `<selectmenu>` element
 
-_Published on 5 May 2022._
+May 31, 2022
 
-[![Thumbnail image for the DevTools customization video](./index-images/customize-devtools.png)](https://www.youtube.com/watch?v=ypRzEBYNptQ)
+[![Thumbnail image for the selectmenu video](./index-images/selectmenu.png)](https://www.youtube.com/watch?v=Ts7jvRyQACY)
 
-Learn how to customize DevTools to fit your needs.
+Styling `<select>` elements has been challenging. The experimental<!-- todo: update.  June 2024: the present repo & MDN don't mention selectmenu --> `<selectmenu>` element promises to overcome the remaining limitations by enabling web developers to style all parts of the element.
```

Comment:
how to word re: was experimental.  blog posts mention selectmenu but not found.
@captainbrosset

Reply:
updated